deploy/configurations.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Template
+parameters:
+- name: ACCOUNT_LIMIT
+  required: true
+- name: ROOT_OU_ID
+  required: true
+- name: BASE_OU_ID
+  required: true
+metadata:
+  name: aws-account-operator-configmap
+objects:
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: aws-account-operator-configmap
+    namespace: aws-account-operator
+  data:
+    account-limit: ${ACCOUNT_LIMIT}
```

Comment:
I would add quotes around `${ACCOUNT_LIMIT}`

Reply:
By adding the quote here do I remove them from the app-interface pr? -> https://gitlab.cee.redhat.com/service/app-interface/-/blob/d3f2ce5db392eb9e55d5ddeeee58a70d986034d2/data/services/osd-operators/cicd/saas/saas-aws-account-operator.yaml#L42-42

Reply:
nope, keep on both